### ISSUE TITLE
EV3: stopping simulator does not stop sound playing

### ISSUE THREAD

**Issue by Blippy2019** [OpenRoberta/openroberta-lab#476]
**Describe the bug**
 - Stopping the simulator has no effect on any sound playing. The sound will continue until it is finished even if the simulation is stopped. When the 'Say' block is used with a long sentence, the robot will finish the sentence even when the simulation is stopped halfway through.

**To Reproduce**
1. Go to EV3 simulation
2. Use a play frequency block and set duration to 10 seconds
3. Start the simulation and stop it immediately
4. See error - sound keeps on playing until 10 seconds pass

**Expected behavior**
 - Sound should stop immediately when simulation is stopped

**Screenshots**
 - 
![image](https://user-images.githubusercontent.com/58810680/71701777-c14af380-2e1f-11ea-916e-e028103e284a.png)
![image](https://user-images.githubusercontent.com/58810680/71701772-b42e0480-2e1f-11ea-81a4-df930e09d743.png)

**Device information**
 - Type: Desktop
 - OS: Windows 10
 - Browser: Chrome

**Comment by boonto** [comment]
Needs testing whether this is the case for the real robot as well.

**Comment by boonto** [comment]
0 brightness works correctly on the real Calliope.

**Comment by boonto** [comment]
EV3dev: The program will stop, but the speech output will still continue in the background.

**Comment by ensonic** [comment]
How did you stop the program on EV3dev - with middle+down buttons?

**Comment by boonto** [comment]
Yes, that should be the same behaviour as the stop button in the simulation. On monday I will try it on lejos and c4ev3 as well and check if it is the same there. If it is the same on every platform the simulation is just accurate :)
